e2e/transactions/transactions_test.go

```diff
@@ -681,16 +691,16 @@ func (s *TransactionsTestSuite) TestDiscardMultipleQueuedTransactions() {
 	}
 
 	// wait for transactions, and discard immediately
-	discardTxs := func(txIDs []common.QueuedTxID) {
-		txIDs = append(txIDs, "invalid-tx-id")
+	discardTxs := func(txIDList []common.QueuedTxID) {
```

Comment:
Not sure how `txIDList` is better than `txIDs`.

Reply:
In outer scope we already have `txIDs` variable so this done to elimitate shadowing.